client/layout/guided-tours/docs/API.md

```diff
@@ -0,0 +1,162 @@
+* `placement`: (string, optional) Placement. Possible values: 'below', 'above', 'beside', 'center', 'middle', 'right'
+* `arrow`: (string, optional) If defined, step will get arrow pointing to a direction. Available: 'top-left', 'top-center', 'top-right',
+'right-top', 'right-middle', 'right-bottom', 'bottom-left', 'bottom-center', 'bottom-right', 'left-top', 'left-middle', 'left-bottom'
+* `style`: (object, optional) Will be used as step's inline style.
+* `when`: (function, optional) This is a redux selector that can prevent step from showing when it evaluates to false. Define `next` prop to tell Guided Tours name of the step it should skip to. If you omit this prop, step will be rendered as expected.
+* `next`: (string, optional) Define this to tell Guided Tours name of the step it should skip to when `when` evaluates to false.
+
+### Example
+
+```jsx
+// single step tour
+<Tour path="/me" name="uselessTour">
+  <Step>
+    <p>This is a page about you.</p>
+    <ButtonRow>
+      <Quit />
+    </ButtonRow>
+  </Step>
+</Tour>
+```
+
+Note that all text content needs to be wrapped in `<p>` so it gets proper styling.
+
+This is just a quick and not very useful tour. You can see more examples in [TUTORIAL.md](TUTORIAL.md) or by exploring existing tours in client/layout/guided-tours/tours.
+
+## ButtonRow
+
+ButtonRow is a React component to display button controls in Step and takes care of their proper styling. Usually used as the last child of Step to contain all available controls.
+
+### Example
+
+```jsx
+<Step>
+  <p>ButtonRow Example</p>
+  <ButtonRow>
+    <Next step="next-step" />
+    <Quit />
+  </ButtonRow>
+</Step>
+```
+
+## Continue
+
+Continue is a React component that you can use in Step to programmatically continue the tour to other step based on user interaction with Calypso. There are currently two ways to declare the condition to continue the tour.
+
+- Binding an `onClick` listener to a DOM node marked as `[data-tip-target]`
+- Redux selector function that evaluates to true in order to advance the tour
+
+### Props
+
+* `step`: (string) Name of the step the tour will advance to.
+* `target`: (string, optional) Name of `[data-tip-target]` that would be watched.
+* `click`: (bool, optional) If true, `onClick` will be listened on `target` DOM node.
+* `hidden`: (bool, optional) If true, this will not render anything in Step, while functionality remains.
+* `icon`: (string, optional) Name of Gridicon to show in custom message.
+* `when`: (function, optional) Redux selector. Once it evaluates to true, tour will advance to `step`.
+
+### Content
+
+- text
+- text + props.icon
+- custom content
+
+### Examples
+
+```jsx
+// continue when user clicks DOM element with html attribute `data-tip-target="my-sites"`
+<Continue step="next-step" click target="my-sites" />
+// continue when Redux selector evaluates to true (in this case after user opens preview)
+<Continue step="next-step" when={ isPreviewShowing } />
```

Comment:
Not sure what the place to say this is, but authors should be aware that `when` will only work if something triggers a re-render of the step and that the proper way to set this up is to add the relevant action types to `actionLog`'s whitelist.